contracts/MetawoodAuction.sol

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "./interfaces/IMetawoodNFT.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+
+contract MetawoodAuction is ERC1155Holder, Ownable, ReentrancyGuard {
+    using SafeMath for uint256;
+    using Counters for Counters.Counter;
+
+    Counters.Counter private _auctionCounter;
+    IMetawoodNFT public metawoodNFT;
+
+    uint256 public bidIncreaseThreshold = 200;
+
+    constructor(address _nftContract) {
```

Comment:
constructor(address _nftContract) => constructor(IMetawoodNFT _nftContract)
So you can directly initialize the variable below

Reply:
:+1: